app/src/main/java/de/westnordost/streetcomplete/data/meta/Abbreviations.kt

```diff
@@ -0,0 +1,86 @@
+package de.westnordost.streetcomplete.data.meta
+
+import java.util.Locale
+import java.util.regex.Matcher
+import java.util.regex.Pattern
```

Comment:
Hmm, is there any reason why to use the Java Regex stuff and not the Kotlin regex stuff?

Reply:
Ooops, I didn't know there was a difference. Android Studio's automatic Java to Kotlin conversion did that and I didn't question it.

Reply:
I've refactored it to use Kotlin `Regex`es in [`4086e04`](https://github.com/streetcomplete/StreetComplete/pull/3715/commits/4086e0438db5b2bf76567e94fe973b5bbaa871a2).

Reply:
Nice, you even made it shorter (and more performant)!